packages/libs/react-ui/src/components/IconButton/IconButton.css.ts

```diff
@@ -55,3 +55,16 @@ export const colorVariants = styleVariants(colors, (color) => {
     sprinkles({ color: `$${color}Contrast`, bg: `$${color}Surface` }),
   ];
 });
+
+export const stateVariants = styleVariants(colors, (color) => {
+  if (['default', 'inverted'].includes(color)) {
+    return [];
+  }
+
+  return [
+    {
+      outlineOffset: '2px',
+      outline: `2px solid ${vars.colors[`$${color as ColorType}Accent`]}`,
+    },
+  ];
+});
```

Comment:
I think there is potentially a cleaner way to implement this. You can make a var for the outline value of "none" and create a class `activeClass` that updates the var to the appropriate outline value. You might also need to make a var for `accentColor` as well, but I think this approach would still be cleaner since these two style variants are tightly coupled

Reply:
like we discussed over in slack, these styles are moved to default color variants styles. only show/hiding outline with non-active class